### PR TITLE
OKTA-461282 Add multi-tenant link to Okta multi-org

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Add multi-tenant solution link to Okta organization > multi-org section
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-461282](https://oktainc.atlassian.net/browse/OKTA-461282)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/concepts/okta-organizations/index.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ In most cases, your company or project has only one Okta organization. Single or
 
 However, in more complex situations, you might need multiple orgs. For example, an organization has external, non-transient workers that require access to Active Directory (AD). The org also has some internal, transient workers who don't require access to AD. Their accounts are created directly in Okta, and they are sent an activation email to set up their password. This activation email would be the same email that external users receive if using a single org. This presents challenges around wording and the branding of the email template.
 
-Multiple orgs allow for complete segregation of internal and external users, and applications and changes made to the internal or external org have no impact on one another. However, multiple orgs add complexity in terms of the number of environments to manage.
+Multiple orgs allow for complete segregation of internal and external users, and applications and changes made to the internal or external org have no impact on one another. However, multiple orgs add complexity in terms of the number of environments to manage. See [Multi-tenant solutions](/docs/concepts/multi-tenancy/) for details.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/concepts/okta-organizations/index.md:54]:
```suggestion
Multiple orgs allow for the complete segregation of internal and external users, and applications and changes made to the internal or external org have no impact on one another. However, multiple orgs add complexity in terms of the number of environments to manage. See [Multi-tenant solutions](/docs/concepts/multi-tenancy/).
```

**Review by christinechoy-okta — APPROVED** [review]
Just a couple of nits, otherwise LGTM!

**Review by robertnicolo-okta — APPROVED** [review]
lgtm

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**